WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/usecases/FetchScanStateUseCase.kt

```diff
@@ -27,32 +26,36 @@ class FetchScanStateUseCase @Inject constructor(
     suspend fun fetchScanState(
         site: SiteModel,
         polling: Boolean = true,
```

Comment:
Minor (🔍): Consider removing this `polling` parameters as it is no longer necessary?

Reply:
In ecbd477